fractions/microprofile/microprofile-fault-tolerance/src/main/java/org/wildfly/swarm/microprofile/faulttolerance/MicroProfileFaultToleranceFraction.java

```diff
@@ -16,27 +16,18 @@
 
 package org.wildfly.swarm.microprofile.faulttolerance;
 
-import org.wildfly.swarm.config.runtime.AttributeDocumentation;
-import org.wildfly.swarm.spi.api.Defaultable;
 import org.wildfly.swarm.spi.api.Fraction;
-import org.wildfly.swarm.spi.api.annotations.Configurable;
+import org.wildfly.swarm.spi.api.Module;
 import org.wildfly.swarm.spi.api.annotations.DeploymentModule;
 import org.wildfly.swarm.spi.api.annotations.DeploymentModule.MetaInfDisposition;
 
 /**
  * @author Antoine Sabot-Durand
  */
-@DeploymentModule(name = "org.wildfly.swarm.microprofile.faulttolerance", export = true, slot = "deployment", metaInf = MetaInfDisposition.IMPORT)
+@DeploymentModule(name = "io.smallrye.faulttolerance", metaInf = MetaInfDisposition.IMPORT, export = true, services = Module.ServiceHandling.IMPORT)
 public class MicroProfileFaultToleranceFraction implements Fraction<MicroProfileFaultToleranceFraction> {
 
     public MicroProfileFaultToleranceFraction() {
     }
 
-    public boolean isSynchronousCircuitBreakerEnabled() {
-        return synchronousCircuitBreaker.get();
-    }
-
-    @AttributeDocumentation("Enable/disable synchronous circuit breaker functionality. If disabled, `CircuitBreaker#successThreshold()` of value greater than 1 is not supported. Moreover, circuit breaker does not necessarily transition from `CLOSED` to `OPEN` immediately when a fault tolerance operation completes. However, applications are encouraged to disable this feature on high-volume circuits.")
-    @Configurable("swarm.microprofile.fault-tolerance.synchronous-circuit-breaker")
-    private Defaultable<Boolean> synchronousCircuitBreaker = Defaultable.bool(true);
```

Comment:
If we remove this it won't be possible to configure this functionality using yaml config files. Are you sure it's a good idea?